gcpy/benchmark/modules/run_1yr_fullchem_benchmark.py

```diff
@@ -388,7 +388,7 @@ def run_benchmark(config, bmk_year_ref, bmk_year_dev):
 
             # Create plots
             print("\nCreating plots for annual mean")
-            bmk.make_benchmark_conc_plots(
+            bmkf.make_benchmark_conc_plots(
```

Comment:
I think this is a typo - `bmk` is still used above

Reply:
Thanks @msulprizio.  I'll fix this.

Reply:
Now fixed in commit cf6fc12